src/jit/compiler.h

```diff
@@ -8817,12 +8817,16 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
     static HelperCallProperties s_helperCallProperties;
 
-#if defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
+#ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
```

Comment:
I think the convention is to use #ifdef if the FEATURE tag is defined in build scripts.  If it is defined in target.h, we are using #if. The reason i have changed it to #ifdef is that throughout Jit sources we are using #if defined(xxx) only if the condition is a compound condition

e.g. #if defined(xx) && defined(yyy)

Whereas #ifdef xxx instead of #if defined(xxx).